doc/source/user/raster_data_model.rst

```diff
@@ -189,6 +189,13 @@ For satellite or aerial imagery the IMAGERY Domain may be present. It depends on
 - CLOUDCOVER: Cloud coverage. The value between 0 - 100 or 999 if not available
 - ACQUISITIONDATETIME: The image acquisition date time in UTC
 
+Starting with GDAL 3.10, there also exists a raster band level IMAGERY metadata domains with the following items:
+
+- CENTRAL_WAVELENGTH_UM: Central Wavelength in micrometers.
+- FWHM_UM: Full-width half-maximum (FWHM) in micrometers.
+
+At time of writing, they are set by the SENTINEL2 and ENVI drivers (if corresponding metadata items are found in the ENVI header)
```

Comment:
Maybe you can say that they are available in other formats, but not populated by default. Otherwise it is easy to think that they are only for these two drivers mentioned here.

Reply:
Also mentioning which function can be used to set them in other drivers could be useful.